third_party/unused_dependency_checker/src/test/BUILD

```diff
@@ -0,0 +1,26 @@
+licenses(["notice"])  # 3-clause BSD
+
+load("//scala:scala.bzl", "scala_junit_test", "scala_test")
+
+scala_test(
+    name = "unused_dependency_checker_test",
+    size = "small",
+    srcs = [
+        "io/bazel/rulesscala/dependencyanalyzer/UnusedDependencyCheckerTest.scala",
+    ],
+    jvm_flags = [
```

Comment:
can you comment about these strings? I don't see them in your code. Seems like you copied the test, but I don't think you need these jvm_flags other than the plugin.jar.location. Am I wrong?

I guess this is the testing fra

Reply:
`-Dguava.jar.location` is not used so removed that.

The other ones are accessed in `TestUtil.scala` and used there and in `UnusedDependencyChecker.scala`


https://github.com/jhnj/rules_scala/blob/johan/dependencyanalyzer-check-unused/third_party/utils/src/test/io/bazel/rulesscala/utils/TestUtil.scala#L58

https://github.com/jhnj/rules_scala/blob/johan/dependencyanalyzer-check-unused/third_party/unused_dependency_checker/src/test/io/bazel/rulesscala/dependencyanalyzer/UnusedDependencyCheckerTest.scala#L39